101-bastion-hub-spoke-vnet/azuredeploy.json

```diff
@@ -0,0 +1,298 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "metadata": {
+                "description": "The location of all resources"
+            },
+            "defaultValue": "[resourceGroup().location]"
+        },
+        "vNetHubName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Hub vNet"
+            },
+            "defaultValue": "vnet-hub"
+        },
+        "vNetSpokeName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Spoke vNet"
+            },
+            "defaultValue": "vnet-spoke"
+        },
+        "vmName": {
+            "type": "string",
+            "metadata": {
+                "description": "The VM Name"
+            },
+            "defaultValue": "vm1"
+        },
+        "vmSize": {
+            "type": "string",
+            "metadata": {
+                "description": "The size of the VM"
+            },
+            "defaultValue": "Standard_A2_v2"
+        },
+        "adminUsername": {
+            "type": "string",
+            "metadata": {
+                "description": "The administrator username"
+            }
+        },
+        "adminPassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The administrator password"
+            }
+        },
+        "storageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the storage account that will be used for boot diagnostics"
+            }
+        },
+        "bastionHostName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the Azure Bastion host"
+            },
+            "defaultValue": "bastion1"
+        }
+    },
+    "functions": [],
+    "variables": {
+        "vNetHubPrefix": "10.0.0.0/16",
+        "subnetBastionName": "AzureBastionSubnet",
+        "subnetBastionPrefix": "10.0.0.0/27",
+        "vNetSpokePrefix": "10.1.0.0/16",
+        "subnetSpokeName": "Subnet-1",
+        "subnetSpokePrefix": "10.1.0.0/24",
+        "publicIPName": "pip-bastion-01",
+        "nsgName": "nsg-subnet-1"
+    },
+    "resources": [{
```

Comment:
format the json (E.g. SHIFT+ALT+F in VS Code) - put the [ and { on separate lines